int16 velocity
---
bool success